cartoframes/data/observatory/repository/constants.py

```diff
@@ -0,0 +1,7 @@
+CATEGORY_FILTER = 'category_id'
+COUNTRY_FILTER = 'country_id'
+DATASET_FILTER = 'dataset_id'
+GEOGRAPHY_FILTER = 'geography_id'
+PROVIDER_FILTER = 'provider_id'
+VARIABLE_FILTER = 'variable_id'
+VARIABLE_GROUP_FILTER = 'variable_group_id'
```

Comment:
At this point we have not defined how this three filters (provider, variable and variable group) would work right? Do them make sense at all?

Reply:
They're used for entity navigation, for example when doing: `Variable.get('some_var').datasets`. Since the behavior is the same than for catalog navigation I used the same filters for consistency.